### PR TITLE
Add button to toggle setting

### PR DESCRIPTION
**Pull request by bhazelwood**
I've added a button to toggle the setting on/off, as going into the addon menu is barely quicker than going to about:config and editing the setting.

I thought this might be useful due to the review I saw on the addon page stating a similar thing.

FYI: this is my first time editing a firefox addon, writing javascript and contributing to GitHub so if I've done anything wrong I'd be happy to fix it and learn!

Screen shot with button

### CODE REVIEW

File: lib/main.js [hunk 1]
```diff
@@ -10,4 +32,4 @@ exports.onUnload = function(reason) {
     //    upgraded
     //    downgraded
 	pref.clearUserPref("layout.css.devPixelsPerPx");
-};
```
Comment (rubo77) [inline lib/main.js]: The script should end with a newline

please add this and maybe amend it to your last commit

Comment (bhazelwood) [inline lib/main.js]: Hi, in my main.js file it does end in a newline, how come it doesn't when I created a pull request?

However, I noticed in your original file it doesn't contain a newline. Did you mean it **shouldn't** have a new line at the end?

Comment (rubo77) [inline lib/main.js]: hmmm, strange, maybe it was my mistake in the first place ;)

great work though! 

I will check it out tomorrow

Comment (bhazelwood) [inline lib/main.js]: Thanks, just trying to get the hang of git, can be a bit confusing in places.

In the split view for this PR, the line comment is on the original main.js file, not the new one.

File: lib/main.js [hunk 2]
```diff
@@ -10,4 +32,4 @@ exports.onUnload = function(reason) {
...
\ No newline at end of file
+};
```
Comment (rubo77) [inline lib/main.js]: seems like you can commit on both sides in the split view ;)

everything is ok with your commit, I will try it out now in firefox....

**Comment by rubo77** [comment]
In github you shoule always create a new branch in your fork to create the mćommit. If you use your master branch for it, then your master will be blocked for this pull request and you would have to create a new banch if you want to catch up eventual other commits in the upstream master.

so always create a branch -> edit your files -> create the pull request from that branch.

But this time I guess it will be all ok, because not much happens in my upstream repo ;)

**Comment by rubo77** [comment]
Thanks a lot for your commit, It is uploaded for review: https://addons.mozilla.org/de/firefox/addon/zoom-menu-elements/versions/?page=1#version-2.0

**Comment by bhazelwood** [comment]
No problem, thanks for the tip about the branches :)

**Pull request merged**